groups/chips-rep-db/ec2.tf

```diff
@@ -40,20 +33,13 @@ module "db_ec2_security_group" {
     }
   ]
 
-  ingress_with_source_security_group_id = [
-    {
-      from_port                = 1521
-      to_port                  = 1522
-      protocol                 = "tcp"
-      description              = "Oracle DB CHIPS OLTP Security Group"
-      source_security_group_id = data.aws_security_group.chips_oltp.id
-    },
+  ingress_with_source_security_group_id = [for group in local.source_security_group_id :
     {
       from_port                = 1521
       to_port                  = 1522
       protocol                 = "tcp"
-      description              = "Oracle DB Staffware Security Group"
-      source_security_group_id = data.aws_security_group.staffware.id
+      description              = "Oracle DB CHIPS Security Group"
```

Comment:
CHIPS -> CHIPS REP

Reply:
Changed to description              = "Oracle DB CHIPS REP Security Group"